advanced/sr_gui_bootloader/src/sr_gui_bootloader/bootloader.py

```diff
@@ -40,21 +35,22 @@ def __init__(self, parent, nb_motors_to_program, prefix):
         self.parent = parent
         self.nb_motors_to_program = nb_motors_to_program
         self.prefix = prefix
+        self.bootloader_service = None
 
     def run(self):
         """
         perform bootloading on the selected motors
         """
         bootloaded_motors = 0
-        firmware_path = self.parent._widget.txt_path.text()
+        firmware_path = self.parent._widget.txt_path.text()  # pylint: disable=W0212
```

Comment:
instead of disabling pylint, it would be better to create a getter for this variable

Reply:
I would do this if it was something we wrote ourselves, but it looks like this comes from qt itself